docs/Installation.md

```diff
@@ -66,6 +66,7 @@ as well as the supported versions of Python:
 | NNCF      | OpenVINO   | PyTorch  | ONNX     | TensorFlow | Python |
 |-----------|------------|----------|----------|------------|--------|
 | `develop` | `2023.3.0` | `2.2.1`  | `1.13.1` | `2.12.0`   | `3.8`  |
```

Comment:
Shouldn't 2.9.0 be added here?

Reply:
See - https://github.com/openvinotoolkit/nncf/pull/2545

Reply:
ok